Add information for Incident Analytics Querying (#11024)
DataDog/documentation: content/en/dashboards/querying/_index.md

If your level of detail is resource or span, some widget types also require you to select a Resource name (`resource`) to narrow the scope of your query.

## Configuring an Incident Analytics graph
Incident Analytics provides you with the following measures:

* Incident Count
* Customer Impact Duration
* Status Active Duration
* Status Stable Duration
* Time to Repair 
* Time to Resolve 

Incident Analytics is currently supported in the following widgets:

* Timeseries
* Top List 
* Query Value 

To configure your graph using Incident Analytics data, follow these steps:

1. [Select your visualization](#select-your-visualization) (same as for Metrics)
2. Select `Incidents` from the data source dropdown menu

> 3. Select a Measure from the yellow dropdown menu

Reviewer: ```suggestion
3. Select a measure from the yellow dropdown menu
```

>      - **Default Statistic:** Count the number of Incidents

Reviewer: ```suggestion
     - **Default Statistic:** Count the number of incidents
```

5. Select an aggregation for the measure
6. (Optional) Select a rollup for the measure

> 7. (Optional) Use the search bar to filter the statistic down to a specific subset of Incidents

Reviewer: ```suggestion
7. (Optional) Use the search bar to filter the statistic down to a specific subset of incidents
```

> 8. (Optional) Select a facet in the pink dropdown menu to break the measure up by a group and select the limit of groups to display 

Reviewer: ```suggestion
8. (Optional) Select a facet in the pink dropdown menu to break the measure up by group and select a limited number of groups to display 
```
Is this what you mean?

> 9. [Title the graph](#create-a-title) (same as for Metrics)

Reviewer: ```suggestion
9. [Title the graph](#create-a-title) (same as for metrics)
```
